### PR TITLE
Changed README so DRUID runs with its most recent updates and provide…

### PR DESCRIPTION
**Pull request by vasthemas**
… more description of datasets used

### CODE REVIEW

**Review by ranikay — APPROVED** [review]
This looks nice Vasanth, thanks for adding this info!

**Review by vhorvath — APPROVED** [review]
Thanks for the additions and making it more organized!

File: README.md [hunk 1]
```diff
@@ -6,59 +6,129 @@ DRUID, or DRUg Indication Discoverer, is an algorithm that identifies drug profi
 
 Install from GitHub using `devtools` as:
 
-```
+```r
 devtools::install_github("diogocamacho/druid")
+devtools::install_github("diogocamacho/cauldron")
```
Comment (vhorvath) [inline README.md:11]: We probably want to change these to the Wyss repository.

```suggestion
devtools::install_github("Wyss/druid")
devtools::install_github("Wyss/cauldron")
```

**Review by vhorvath — COMMENTED** [review]
I just noticed a tiny change that might make sense since we work on our own version.

File: README.md (hunk 1 above)
Comment (vasthemas) [reply README.md]: Thank you for the suggestion that makes more sense.

**Pull request merged**